### PR TITLE
Added new cloudi.1.7.1

### PR DESCRIPTION
**Pull request by okeuday**
This is the release version of the OCaml CloudI API to make OCaml development a bit easier with CloudI (http://cloudi.org)

### CODE REVIEW

**Comment by okeuday** [comment]
It appears that the checks assume the package already exists, so it looks like there is a CI problem here.

File: packages/cloudi/cloudi.1.7.1/cloudi.opam [hunk 1]
```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
```
Comment (samoht) [inline packages/cloudi/cloudi.1.7.1/cloudi.opam]: this file shoud be called `opam`, not `cloudi.opam`

**Comment by okeuday** [comment]
Thank you for catching this problem with the package opam file.  I am still getting use to opam.

**Comment by samoht** [comment]
No pb, glad to see that everything is green now :-)

**Pull request merged**